flashinfer_bench/data/validate.py

```
@@ -0,0 +1,1245 @@
+    """Load a pydantic model from a JSON file. Returns (instance, None) or (None, error_str)."""
+    try:
+        return model_class.model_validate_json(path.read_text()), None
+    except (ValidationError, Exception) as exc:
+        return None, str(exc)
+
+
+def _load_jsonl(model_class, path: Path):
+    """Load a list of pydantic models from JSONL. Returns (list, None) or (None, error_str)."""
+    items = []
+    try:
+        for line_number, line in enumerate(path.read_text().splitlines(), 1):
```

Ruff (0.15.9)

[warning] Loop control variable `line_number` not used within loop body

(B007)